### DISCUSSION TITLE
Organization/client(application) level user fields and roles

### DISCUSSION THREAD

**Discussion by tyzhnenko** [fief-dev/fief#466] [Ideas]
Support user fields which belong to some organization or a client.

Common fields like first and last names can be global. On the other hand, some organization/client may have additional fields which are extra or redundant for other organizations.

The same idea for roles/permissions.

P.S. Maybe it makes sense to discard global wide fields/roles/permissions.

**Comment by fief-bailiff** [comment]
Hail, @tyzhnenko 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!
